presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorMetadata.java

```diff
@@ -67,6 +67,14 @@ default boolean schemaExists(ConnectorSession session, String schemaName)
         return listSchemaNames(session).contains(schemaName);
     }
 
+    /**
+     * Check if connector is providing legacy connnectorTableLayout
+     */
+    default boolean useLegacyLayout()
```

Comment:
Can we introduce this function together with the logic using it?

Reply:
Yes, this is added just as placeholder, will remove later.